x-pack/plugins/infra/public/components/metrics_explorer/chart_context_menu.test.tsx

```diff
@@ -82,4 +93,41 @@ describe('MetricsExplorerChartContextMenu', () => {
     expect(menuItems.length).toBe(1);
     expect(menuItems.at(0).prop('disabled')).toBeTruthy();
   });
+
+  it('should not display "Open in Visualize" when unavailble in uiCapabilities', async () => {
+    const customUICapabilities = { ...uiCapabilities, visualize: { show: false } };
+    const onFilter = jest.fn().mockImplementation((query: string) => void 0);
+    const component = mountWithIntl(
+      <MetricsExplorerChartContextMenu
+        timeRange={timeRange}
+        source={source}
+        series={series}
+        options={options}
+        onFilter={onFilter}
+        uiCapabilities={customUICapabilities}
+      />
+    );
+
+    component.find('button').simulate('click');
+    const menuItems = component.find('button.euiContextMenuItem');
+    expect(menuItems.length).toBe(1);
+    expect(menuItems.at(0).text()).toBe('Add Filter');
+  });
+
+  it('should not display anything', async () => {
```

Comment:
Is there any way we can make this test more meaningful? When should it not display what? And how is that different from the previous test case?

Reply:
I'm waiting for #37136 to be merged for before I make any changes to these tests. But I will change the test to make more sense.